GAE/src/org/waterforpeople/mapping/app/web/RawDataRestServlet.java

```diff
@@ -180,10 +188,12 @@ protected RestResponse handleRequest(RestRequest req) throws Exception {
 
                     answer.setValue(response);
                     answer.setType(type);
+                    answer.setIteration(iteration);
```

Comment:
I was just wondering if strictly speaking `iteration` shouldn't only be set when `answer == null`.  In fact thinking about it, perhaps `type` should also not be changed because this creates possibilities for inconsistencies..